contrib/executor/init/pkg/runner/runner.go

```diff
@@ -87,18 +87,18 @@ func (r *InitRunner) Run(ctx context.Context, execution testkube.Execution) (res
 		postRunScript := shebang
 
 		if execution.PreRunScript != "" {
-			entrypoint += strconv.Quote(filepath.Join(r.Params.DataDir, preRunScriptName)) + "\n"
+			entrypoint += "source " + strconv.Quote(filepath.Join(r.Params.DataDir, preRunScriptName)) + "\n"
```

Comment:
`. file.sh` is slightly more portable across shells than `source file.sh`, but nowadays `source` should be in most places anyway

Reply:
I was planning to test it today, might be we need a flag to use source

Reply:
Will add a flag for flexibility

Reply:
`.` is rather enough - in shells where `source` is implemented, rather always it's alias for `.`